src/sentry/migrations/0618_monitors_migrate_is_muted.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.23 on 2023-12-08 20:47
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def migrate_is_muted(apps, schema_editor):
+    Monitor = apps.get_model("sentry", "Monitor")
+    query = Monitor.objects.filter(status=1)
+
+    # Query muted monitors
+    for monitor in RangeQuerySetWrapperWithProgressBar(query):
+        monitor.is_muted = True
+        monitor.save()
```

Comment:
`UPDATE ... SET is_muted = true WHERE status = MonitorEnvironmentStatus.MUTED`?

Reply:
We typically do these with a `RangeQuerySetWrapperWithProgressBar` to avoid locking the database and stampeding I think.